### PR TITLE
AMBR-1115 Use log4j for all logging

### PR DESCRIPTION
**Pull request by egh**
Allow configuration with LOG_LEVEL environment variable.

### CODE REVIEW

**Comment by egh** [comment]
@evarghese ping

File: src/main/java/org/plos/repo/config/SpringConfig.java [hunk 1]
```diff
@@ -22,45 +22,55 @@
...
 @Configuration
 @EnableTransactionManagement
 public class SpringConfig {
 
-  private static final Logger log = LoggerFactory.getLogger(SpringConfig.class);
+  private static final Logger log = LogManager.getLogger(SpringConfig.class);
 
   @Bean
   public RepoInfoService repoInfoService() {
+    log.debug("YOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO");
```
Comment (egh) [inline src/main/java/org/plos/repo/config/SpringConfig.java]: uh, I'll remove this

Comment (sgatchell) [reply src/main/java/org/plos/repo/config/SpringConfig.java]: :100::o2::o2::o2::o2::o2::o2:

Comment (egh) [reply src/main/java/org/plos/repo/config/SpringConfig.java]: now you know what my debug log lines all look like.

**Review by sgatchell — APPROVED** [review]
Looks great, builds great.  :shipit:

**Pull request merged**